arches/management/commands/updateproject.py

```diff
@@ -15,6 +19,13 @@ def handle(self, *args, **options):
             self.update_to_v7()
 
     def update_to_v7(self):
+        # copy webpack config files to project
+        project_webpack_path = settings.APP_ROOT + "/webpack/"
```

Comment:
I think `os.path.join` should probably be used here and on line 26 to make this command Windows safe.

Reply:
good eye